data/completion/complete.sh

```diff
@@ -116,7 +116,7 @@ else
     _complete_from_snap_maybe() {
         local etel=snap/core/current/usr/lib/snapd/etelpmoc.sh
         # catch /snap/bin and /var/lib/snapd/snap/bin
-        if [[ "$(which "$1")" =~ /snap/bin/ && ( -e "/var/lib/snapd/$etel" || -e "/$etel" ) ]]; then
+        if [[ "$(command -v "$1")" =~ /snap/bin/ && ( -e "/var/lib/snapd/$etel" || -e "/$etel" ) ]]; then
```

Comment:
the problem with this is that if you have an *bash* alias to a command, it'll match this
```sh
~$ alias boo=/snap/bin/http
~$ command -v boo
alias boo='/snap/bin/http'
~$ [[ "$( command -v boo )" =~ /snap/bin/ ]] && echo YES
YES
```
but the completer will have no idea how to handle it.

Reply:
Hmm, hmm, shall I drop this patch then?

Reply:
Done now